### DISCUSSION TITLE
Linux port

### DISCUSSION THREAD

**Discussion by gibarel1** [Y0URD34TH/Project-GLD#24] [Ideas]
How hard would it be to port this tool (mainly the search/download aspect) to Linux? There are library manager on Linux such as lutris, but it only searches for legit sources like steam, gog and epic.

**Comment by Y0URD34TH** [comment]
would be easy to make a port for linux, the only problem is related to testing (we dont have enouth people to test), but im working on the port for linux and mac, it is very likely that the port will come in the next update

**Reply by gibarel1** [reply]
I'd happily help you test it on linux, and I'm sure the folks at reddit.com/r/LinuxCrackSupport would gladly help as well. On that topic, another interesting thing that could be in the linux client is sourcing zamunda for Linuxrulez repacks, they come with all the dependencies for the game to work on linux.

**Reply by Y0URD34TH** [reply]
are you already on discord?

**Reply by gibarel1** [reply]
I've just entered it

**Comment by argonptg** [comment]
> another interesting thing that could be in the linux client is sourcing zamunda for Linuxrulez repacks, they come with all the dependencies for the game to work on linux.

Does zamunda have like cloudflare protection? If not then making a script for it might be easy

**Reply by gibarel1** [reply]
I'm not sure, I know it's blocked on some countries so it might have.
